packages/shadycss/src/scoping-shim.js

```diff
@@ -240,6 +241,29 @@ export default class ScopingShim {
     }
     this._customStyleInterface['enqueued'] = false;
   }
+  /**
+   * Validate ordering of custom styles for Custom Property shim
+   * @param {!Array<!CustomStyleProvider>} customStyles
+   */
+  _validateCustomStyleOrder(customStyles) {
```

Comment:
This function seems to be establishing the order itself or repairing it rather than validating - maybe a different name?

Reply:
Yeah, I'll just rename it to `_reorderCustomStylesRules`

Reply:
Done. PTAL